lib/datasource/github-releases/index.ts

```diff
@@ -1,5 +1,5 @@
 import * as packageCache from '../../util/cache/package';
-import { GithubHttp } from '../../util/http/github';
+import { GithubHttp } from '../../util/http/datasource';
```

Comment:
Not sure this will work. Won't we need the github wrapper? Alternatives:
- skip mem clearing for the platform-related datasources, or
- have them manually embed the bucket in their requests

Reply:
In `util/http/datasource.ts` I've created 3 wrappers for `Http`, `GithubHttp` and `GitlabHttp` which are using same `datasource` bucket. I still need to test it on real repo, but I believe it should work.

Reply:
Oh, I missed the `GithubHttp` part being copied over